speculative_decoding/packages/helpers.py

```diff
@@ -66,7 +68,7 @@ class _TimingManager(object):
     def __init__(self):
         self._timings = collections.defaultdict(_Timing)
         self._order = []
-        self._total = None
+        self.total_section_name = None
```

Comment:
nit: Is this meant to be public? We generally keep all member variable private and create a public property function to expose.